ingress-node-firewall/v1alpha1/ingressnodefirewall_types_methods.go

```diff
@@ -0,0 +1,58 @@
+package v1alpha1
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func (p IngressNodeFirewallProtoRule) IsRange() bool {
```

Comment:
Should these functions be removed now the webhooks are in a different package?

Reply:
they are used with ports